src/monitoring_service/handlers.py

```diff
@@ -176,7 +178,7 @@ def channel_closed_event_handler(event: Event, context: Context) -> None:
             "Settle period timeout is in the past, skipping",
             token_network_address=event.token_network_address,
             identifier=channel.identifier,
-            settle_until=settle_until,
+            settle_until=settleable_after,
```

Comment:
Is this correct? Assigning a variable `_after` to a `_until` raises some flags

Reply:
This was a good catch. Especially because it was hint to pure naming in the above code. While this was still from the old code, it was simply not good and we took the opportunity to improve here. For the better naming I discussed that with Fernando and Paul. The commit addressing this got pushed.

Reply:
✔️